packages/snap-store-mobx/src/types.ts

```diff
@@ -54,6 +54,13 @@ export type SearchStoreConfig = StoreConfig & {
 				[field: string]: FacetStoreConfig;
 			};
 		};
+		filters?: {
+			hierarchy?: {
+				showInSummary?: boolean;
```

Comment:
I wonder if this should just be `enabled` to match some of the other configurations...?  🤔

Reply:
sure